java/code/src/com/redhat/rhn/manager/kickstart/cobbler/test/CobblerEnableBootstrapCommandTest.java

```diff
@@ -46,6 +48,13 @@
  */
 public class CobblerEnableBootstrapCommandTest extends BaseTestCaseWithUser {
 
+    @Override
+    @BeforeEach
+    public void setUp() throws Exception {
+        super.setUp();
+        HibernateFactory.getSession().createQuery("DELETE FROM ActivationKey").executeUpdate();
```

Comment:
Why exactly do we need these? Do these leak from some other test?

Reply:
oh ok, so it's caused by your addition. In that case, I would suggest overriding `tearDown()` instead of `setUp()`